src/table/utils/__tests__/copy-utils.spec.ts

```diff
@@ -4,26 +4,94 @@ describe('copyCellValue: ', () => {
   let evt: {
     target: HTMLElement;
   };
+  let writeMock: (arg: string) => void;
+  let elementClass: string;
+  let closestElementClass: string;
+
+  const classList = {
+    contains: (className: string) => className === elementClass,
+  };
+
+  const closest = (className: string) => {
+    className === closestElementClass;
+    return {
+      querySelector: jest.fn(() => ({
+        textContent: 'svg text',
+      })),
+    };
+  };
+
   beforeEach(() => {
-    console.log = jest.fn();
+    elementClass = '';
+    closestElementClass = '';
+    evt = {
+      target: {
+        closest,
+        querySelector: jest.fn(() => ({
+          textContent: 'head text',
+        })),
+        classList,
+      } as unknown as HTMLElement,
+    };
+    jest.spyOn(console, 'log');
+    writeMock = jest.fn();
+    Object.assign(navigator, {
+      clipboard: { writeText: writeMock },
+    });
   });
 
-  beforeEach(() => {
-    jest.clearAllMocks();
+  afterEach(() => jest.clearAllMocks());
+
+  it('should call clipboard.writeText but not console log for a table cell', async () => {
+    elementClass = 'sn-table-cell';
+    await copyCellValue(evt);
+    expect(global.console.log).toHaveBeenCalledTimes(0);
+    expect(writeMock).toHaveBeenCalledTimes(1);
+    expect(writeMock).toHaveBeenCalledWith('head text');
+  });
+
+  it('should call clipboard.writeText but not console log for a non-table cell', async () => {
+    elementClass = 'sn-table-head-cell';
+    closestElementClass = 'sn-table-cell';
```

Comment:
this is not a real case. If it has sn-table-head-cell class, it will have sn-table-cell class as well. You can just set elementClass to '' or 'someClass' if you want